samples/powerstore_v230_k8s_124.yaml

```diff
@@ -63,17 +63,28 @@ spec:
       # Allowed values: map of key-value pairs
       # Default value: None
       # Examples:
-      #   node-role.kubernetes.io/control-plane: ""
+      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
```

Comment:
Please remove all samples related to `master` in this file as it is not supported in K8s1.24

Reply:
done